src/wp-includes/l10n.php

```diff
@@ -111,6 +111,25 @@ function get_user_locale( $user = 0 ) {
 	return $locale ? $locale : get_locale();
 }
 
+/**
+ * Retrieves the locale of a user that is associated with the admin_email.
+ *
+ * If there is no user associated with the admin_email, defaults to the site_language
+ *
+ * @since 6.7.0
+ *
+ * @return string The locale of the admin.
+ */
+function get_admin_locale() {
```

Comment:
Note that this name clashes with https://core.trac.wordpress.org/ticket/49971

Reply:
Not just the name, but the whole idea is also very similar :)

Reply:
Yes very similar, but instead of adding a new "Admin language" setting, it's trying to figure out the language by looking for a user.

Reply:
Now this function is no longer needed :-)